spec/test-utils/webrtc.ts

```diff
@@ -481,6 +481,9 @@ export class MockCallMatrixClient extends TypedEventEmitter<EmittedEvents, Emitt
     public getUserId(): string {
         return this.userId;
     }
+    public getSafeUserId(): string {
+        return this.userId;
+    }
```

Comment:
This is a stub for MatrixClient, `getUserId` actually returns `string | null`
![image](https://github.com/user-attachments/assets/2d421898-4a06-4629-a729-f3e646e4c951)

The code under test was changed to use `getSafeUserId` to avoid getting `null` and thus the test must also make this method available so it doesn't end up throwing an exception